samples/subsys/video/tcpserversink/src/main.c

```diff
@@ -15,12 +15,6 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(main);
 
-#if defined(CONFIG_VIDEO_MCUX_CSI)
-#define VIDEO_CAPTURE_DEV DT_LABEL(DT_INST(0, nxp_imx_csi))
-#else
-#define VIDEO_CAPTURE_DEV "VIDEO_SW_GENERATOR"
-#endif
```

Comment:
I guess this was falling back to `drivers/video/video_sw_generator.c` (a non-DT device, should likely be converted to DT...) if `!defined(CONFIG_VIDEO_MCUX_CSI)`, is this keeping the same behavior?

Reply:
Yeah, video_sw_generator.c should probably be converted to DT, but out of scope for this PR :).

Reply:
but this sample won't fallback to the soft version if `!CONFIG_VIDEO_MCUX_CSI`: we're changing its behavior.

Reply:
True, but the only platforms that the sample supports in sample.yaml are one's that have `CONFIG_VIDEO_MCUX_CSI`.

Reply:
@gmarull ping

Reply:
thanks for the explanation, makes sense now.